### ISSUE TITLE
Change ports to ones not already assigned by IANA

### ISSUE THREAD

**Issue by metacollin** [Orthogonal-Systems/Origin#9]
I noticed that by default, Origin's port usage is as follows (comments were added by me):

```
register_port       = 5556 // UDP, TCP registered with IANA for FreeCiv
measure_port        = 5557 //UDP Reserved, TCP registered with IANA for Sandlab FARENET
alert_port          = 5558 
read_port           = 5559 
pub_port            = 5560
json_register_port  = 5566 //UDP Reserved, TCP registered with IANA for Westec Connect
json_measure_port   = 5567 // UDP,TCP IANA: DOF Protocol Stack Multicast/Secure Transport 
```

It would be ill advised to use ports that have official registration with the IANA.  See:

[Port 5556](https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml?search=5556)
[Port 5557](https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml?search=5557)
[Port 5566](https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml?search=5566)
[Port 5567](https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml?search=5567)

However, ports 5558-5564 are unassigned and not reserved on either UDP or TCP.   This conveniently is the perfectly sized window of free ports for Origin.  The ports could simply be:

```
register_port       = 5558
measure_port        = 5559
alert_port          = 5560
read_port           = 5561
pub_port            = 5562
json_register_port  = 5563
json_measure_port   = 5564
```

Another solution is to just translate the original ports by +50.  Doing that doesn't overlap with any registered ports, but maintains the relative port positions:

```
register_port       = 5606
measure_port        = 5607
alert_port          = 5608
read_port           = 5609
pub_port            = 5010
json_register_port  = 5616
json_measure_port   = 5617
```

**Comment by mfe5003** [comment]
Changing to 5558-5564 should be fine.
Do you want to push up the change?

**Comment by mfe5003** [comment]
fixed in commit: 7d491e99ebab921dba404b4bce7069f5086be43e
